tests/Unit/Validator/ValidAddProductBundleToCartCommandValidatorTest.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+ */
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusProductBundlePlugin\Unit\Validator;
+
+use BitBag\SyliusProductBundlePlugin\Command\AddProductBundleToCartCommand;
+use BitBag\SyliusProductBundlePlugin\Entity\ProductBundleInterface;
+use BitBag\SyliusProductBundlePlugin\Validator\ValidAddProductBundleToCartCommand;
+use BitBag\SyliusProductBundlePlugin\Validator\ValidAddProductBundleToCartCommandValidator;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Persistence\ObjectRepository;
+use PHPUnit\Framework\MockObject\MockObject;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Inventory\Checker\AvailabilityCheckerInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+use Webmozart\Assert\InvalidArgumentException;
+
+/**
+ * @property AddProductBundleToCartCommand $value
+ * @property ValidAddProductBundleToCartCommand $constraint
+ */
+final class ValidAddProductBundleToCartCommandValidatorTest extends ConstraintValidatorTestCase
+{
+    /** @var ObjectRepository|mixed|MockObject */
+    private $productBundleRepository;
+
+    /** @var mixed|MockObject|OrderRepositoryInterface */
+    private $orderRepository;
+
+    /** @var mixed|MockObject|AvailabilityCheckerInterface */
+    private $availabilityChecker;
+
+    protected function setUp(): void
+    {
+        $this->productBundleRepository = $this->createMock(ObjectRepository::class);
+        $this->orderRepository = $this->createMock(OrderRepositoryInterface::class);
+        $this->availabilityChecker = $this->createMock(AvailabilityCheckerInterface::class);
+        $this->value = new AddProductBundleToCartCommand();
```

Comment:
Value and constraint are validation arguments so there's no need to specify them here. (you don't use this value later, by the way)